src/map/status.hpp

```diff
@@ -1307,7 +1307,6 @@ enum sc_type : int16 {
 	SC_PERIOD_RECEIVEITEM_2ND,
 	SC_PERIOD_PLUSEXP_2ND,
 
-	SC_EXTREMITYFIST2,
```

Comment:
I thought this was already commented on so it's harder to load the rest of the code, but we need to be careful about erasing SC in the middle since this is the array index stored in SQL. If we need to remove it we should provide a SQL upgrade script to adjust index values above this -1.

Reply:
See my explanation in the reviewer notes. Only two SCs come after it and those are not stored beyond server restart.

Reply:
How about 
`SC_EXTREMITYFIST2,	// unused`

Reply:
If you guys are concerned about this, I'll just do it the normal way (give the next one a fixed number). Just wanted the code to be more clean and we kept moving this SC since like a month ago anyway, so I didn't think it will hurt.

Reply:
Fixed